api/millenium_patron.py

```diff
@@ -63,6 +64,18 @@ class MilleniumPatronAPI(BasicAuthenticationProvider, XMLParser):
     AUTHENTICATION_MODES = [
         PIN_AUTHENTICATION_MODE, FAMILY_NAME_AUTHENTICATION_MODE
     ]
+
+    SETTINGS = [
+        { "key": VERIFY_CERTIFICATE, "label": _("Verify Certificate"), "optional": True },
```

Comment:
I'd like to remove this setting, or change it to "Don't verify certificate", so it makes more sense if you leave it blank.